fdbclient/command_runner.go

```diff
@@ -37,28 +40,61 @@ type realCommandRunner struct {
 	log logr.Logger
 }
 
+// getEnvironmentVariablesWithoutBlacklisted returns the current environment variables for the new process with some
+// FDB specific variables filtered out to ensure we don't set any variables that could change the behaviour of fdbcli or
+// the other fdb tools.
+func getEnvironmentVariablesWithoutExcludedFdbEnv() []string {
+	blackListedEnvironmentVariables := map[string]fdbv1beta2.None{
```

Comment:
I think we generally try to avoid the term "black list" for these kind of use cases. Can we go with something like "blocked" or "excluded" instead?

Reply:
Yes, I adjusted the method name but forgot about the variable name 🤦